interface/patient_file/summary/demographics_full.php

```diff
@@ -378,7 +379,21 @@ function policykeyup(e) {
 </head>
 
 <body class="body_top">
-<form action='demographics_save.php' name='demographics_form' id="DEM" method='post' onsubmit="submitme(<?php echo $GLOBALS['new_validate'] ? 1 : 0;?>,event,'DEM')">
+
+<?php //Check if new patient is added to hooks
+//this will hold the javascript submit me param new_validate;
+$new_validate = $GLOBALS['new_validate'] ? 1 : 0;
+/*
+ * 	you can use the hook to check the validation of the patient from here
+if($GLOBALS['full_new_patient_form'] == '4')//use hook of patient validation = 4
+{
+	//$hook = checkIfPatientValidationHookIsActive();
+
+}
+
+*/
```

Comment:
This form didn't have any logic for calling patient validation check so now I have added this functionallity .To use your own logic I am sending a parameter to the controller where you can write your own logic on this parameters  "closeBeforeOpening=1" , also I am sending all the params that are required like written  in new_compreansive .